src/plugin/plugin.rs

```diff
@@ -241,3 +284,66 @@ where
         }
     }
 }
+
+/// Specifies a default configuration for the default `RapierContext`
+///
+/// If [`None`], no world will be initialized, you are responsible of creating and maintaining
+/// a [`RapierContext`] before creating any rapier entities (rigidbodies, colliders, joints),
+/// and as long as any [`RapierContextEntityLink`] has a reference to its [`RapierContext`].
```

Comment:
I think this last comment line should be rephrased a bit, I have trouble understanding it.